src/parallax/first/index.tsx

```diff
@@ -0,0 +1,22 @@
+import "./index.css";
```

Comment:
question: Are you aware that this makes the CSS global? You may want to consider using cssModules instead to not conflict with upcoming CSS classes

Reply:
I added sass with modules, tho now I kinda wonder why I added sass at all but u know, might use it later?

Reply:
As long as it's scoped CSS anything works really ;) so don't overthink about it, it's all CSS at the end of the day.